.github/workflows/ecosystem-ci.yml

```diff
@@ -21,7 +37,9 @@ jobs:
           node-version: 16
       - run: npm i -g pnpm
       - run: pnpm i --frozen-lockfile
-      - run: node index.js --viteBuildOnly # checkout, install and build vite
+      - name: build branch
+        if: ${{github.event.inputs.}}
+        run: node ecosystem-ci.js build-vite --${{ github.event.inputs.refType || github.event.client_payload.refType || 'branch' }} ${{ github.event.inputs.ref || github.event.client_payload.ref || 'main' }}
```

Comment:
not exactly sure if chaining inputs works this way. Is there a unified api for action inputs?